index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>Javascript test</h1>
+<script>var x = document.getElementById("Button");</script>
```

Comment:
I see that you wanted to use the proper way of interacting with the HTML document with Javascript. What ou should have done is to use `document.createElement("button");`. After that, you would have had a **reference** to the new button in the `x` variable, to which you would have been able to add an event listener.

We are undestanding of the fact that you might have fallen behind with the curriculum, but sometimes the solution is just one Google search away. Using the actual exercise prompt, typing `javascript create a button`  into Google, the top result is [this page](https://sebhastian.com/javascript-create-button/), which literally contains a part of the solution. You need to practice using the internet as an aid, since that is a necessary skill of the modern developer.